Purchases/Purchasing/Purchases.swift

```diff
@@ -1081,43 +1081,43 @@ public extension Purchases {
     }
 
     /**
-     * Initiates a purchase of a ``StoreProduct`` with a ``StoreProductDiscount``.
+     * Initiates a purchase of a ``StoreProduct`` with a ``PromotionalOffer``.
      *
      * Use this function if you are not using the Offerings system to purchase a ``StoreProduct`` with an
-     * applied ``StoreProductDiscount``.
-     * If you are using the Offerings system, use ``Purchases/purchase(package:discount:completion:)`` instead.
+     * applied ``PromotionalOffer``.
+     * If you are using the Offerings system, use ``Purchases/purchase(package:promotionalOffer:completion:)`` instead.
      *
      * - Important: Call this method when a user has decided to purchase a product with an applied discount.
      * Only call this in direct response to user input.
      *
-     * - Important: Before calling this method, check that the user is eligible for the discount
-     * by calling ``checkPromotionalDiscountEligibility(forProductDiscount:product:)``. 
-     *
      * From here ``Purchases`` will handle the purchase with `StoreKit` and call the ``PurchaseCompletedBlock``.
      *
      * - Note: You do not need to finish the transaction yourself in the completion callback, Purchases will handle
      * this for you.
      *
      * - Parameter product: The ``StoreProduct`` the user intends to purchase.
-     * - Parameter discount: The ``StoreProductDiscount`` to apply to the purchase.
+     * - Parameter promotionalOffer: The ``PromotionalOffer`` to apply to the purchase.
```

Comment:
nitpick: we could add a note here telling folks to obtain this either through `eligibileDiscounts` or through `getPromotionalOffer`

Reply:
and have them as related symbols